guessthecard-index.html

```diff
@@ -22,18 +22,20 @@ <h1>Guess the Card</h1> <!--Heading with my name-->
     <!--This is the div where the images will be displayed during the game-->
     <div id = 'mainDiv'>
       <div class="sideByside" id="cardOne">        
-      <img id = "displayingImage1">
+      <img id = "displayingImage1" alt = "false">
```

Comment:
I would recomment to not do a " " between equal signs
```suggestion
      <img id="displayingImage1" alt="false">
```

Reply:
Fixed.
I think i was going with the school days of writing it with spaces :smile: